src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(action => {
```

Comment:
Fix previous comment about destructuring
```suggestion
  actions.forEach({ type,  extraData, keysToRemove }=> {
```